eslint/babel-eslint-parser/package.json

```diff
@@ -1,9 +1,12 @@
 {
-  "name": "babel-eslint",
-  "version": "11.0.0-beta.0",
+  "name": "@babel/eslint-parser",
```

Comment:
I would prefer to defer the rebranding until babel v8, and at that time we can align the `@babel/eslint-parser`'s version to `@babel/core`.

Reply:
We could already align it with 7.8.0

Reply:
It will make people confused, things like when one should use `babel-eslint` and `@babel/eslint-parser`. If we rebrand them in v8, it's more clear: `@babel/eslint-parser` 8.0 works with `@babel/core` 8.0, all the `babel-eslint` supports up to `@babel/core` 7.x

Reply:
I will update the PR tomorrow morning

cc @kaicataldo @existentialism @hzoo

Reply:
I'm in agreement with @nicolo-ribaudo. I actually like that we'll be publishing a different package with the v11 features, because there's already been a lot of confusion around it. Historically, it's always been a challenge to denote the compatibility of a specific version of babel-eslint with Babel and ESLint, and I think this name change will go a long way in solving it on Babel's end.

I also think that releasing babel-eslint v11 (which has a major breaking change in the configuration-aware parsing), forcing users to upgrade, and then releasing @babel/eslint-parser v7.8 a little while later, forcing another migration, is actually more burden on the end user than it would be to just upgrade once.

babel-eslint has been incomplete/buggy for a long time now, and I'm viewing this move as a fresh start to make it more of a first class citizen. Can we just say that @babel/eslint-parser is compatible with whatever the first version we release it in (whether that's 7.8 or 8.0), mark babel-eslint as deprecated, and just guarantee @babel/eslint-parser works from here on out? Old versions won't be any worse than they are now, but hopefully going forward the versioning will make it clear what version of @babel/eslint-* packages one should be using with a particular version of Babel.

All that being said, I'm new around here, and will definitely defer to team consensus 😄